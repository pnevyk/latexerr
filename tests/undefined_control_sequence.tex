\documentclass{article}

\begin{document}
\foo
\end{document}
