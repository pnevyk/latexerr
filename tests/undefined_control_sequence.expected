File: undefined_control_sequence.tex

Error on line 4: Unknown command \foo.
